keyboards/latin60rgb/info.json

```
@@ -0,0 +1,73 @@
+{
+  "keyboard_name": "latin60rgb",
+  "url": "",
+  "maintainer": "latincompass",
+  "width": 14,
+  "height": 5,
+  "layouts": {
+    "LAYOUT": {
```

This needs to match the layout macro name used by the keyboard. 
```suggestion
    "LAYOUT_60_latin_hhkb": {
```